### ISSUE TITLE
Normal LengthvsQualityScatterPlot seems to be using logged read lengths

### ISSUE THREAD

**Issue by genomics-kl** [wdecoster/NanoPlot#167]
The normal scatterplots look very similar to the scatter plots labelled as logged though not identical. The numbers on the X axis for the normal scatter plots also don't make any sense, ranging from 1 to 4.

Version: Nanoplot 1.28.1
Invocation:
```
NanoPlot -t {threads} \
        --outdir {params.outdir} \
        --prefix {params.outprefix} \
        --loglength \
        --fastq_rich {input} \
        2>{log.stderr} 1>{log.stdout}
```

///###
**Not logged**
![sample1_LengthvsQualityScatterPlot_dot](https://user-images.githubusercontent.com/45537554/71735902-ac6d6080-2e1d-11ea-8675-ddd67c20202e.png)
![sample1_LengthvsQualityScatterPlot_kde](https://user-images.githubusercontent.com/45537554/71735904-ac6d6080-2e1d-11ea-87cd-7cbe0995d713.png)

///###
**Log transformed read lengths**
![sample1_LengthvsQualityScatterPlot_loglength_dot](https://user-images.githubusercontent.com/45537554/71735905-ac6d6080-2e1d-11ea-9602-2c4ec568adc1.png)
![sample1_LengthvsQualityScatterPlot_loglength_kde](https://user-images.githubusercontent.com/45537554/71735906-ac6d6080-2e1d-11ea-881e-81ed141462fe.png)

**Comment by wdecoster** [comment]
Thanks for reporting this, that definitely looks off. I think I made a mistake in the latest changes. I'll fix this next week.

**Comment by wdecoster** [comment]
This issue should be solved in NanoPlot v1.28.2, which is now available on PyPI!

Please let me know if you encounter something else unexpected, or have feature requests.

**Comment by genomics-kl** [comment]
Thanks! Will check out the updated version.
